src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -286,5 +287,21 @@ public boolean equals(Object other) {
                     && getInterviewStatus().equals(e.getInterviewStatus())
                     && getAvailability().equals(e.getAvailability());
         }
+
+
+        /**
+         * Allows the {@code InterviewStatus} to be triggered by ApplicationStatus.
+         */
+        public void triggerInterviewStatus(ApplicationStatus applicationStatus) {
+            String acceptedTrigger = "Accepted";
+            String rejectedTrigger = "Rejected";
+            String executedTrigger = "Completed";
+            if (getApplicationStatus().isPresent()) {
+                if (applicationStatus.toString().equals(acceptedTrigger)
+                        || applicationStatus.toString().equals(rejectedTrigger)) {
+                    setInterviewStatus(new InterviewStatus(executedTrigger));
```

Comment:
WJ, could probably import and re-use the static constants declared in `ApplicationStatus` and `InterviewStatus` classes?

```suggestion
                if (applicationStatus.toString().equals(ACCEPTED_STATUS)
                        || applicationStatus.toString().equals(REJECTED_STATUS)) {
                    setInterviewStatus(new InterviewStatus(COMPLETED_STATUS));
```

Reply:
Mm right I will make the changes. I initially thought of localising the variables because it allows anyone to change the trigger easily (eg. if I want rejected to be removed as a trigger, it can be changed easily from this method). But sure I'll standardise across all the methods